.github/workflows/ci-java.yml

```diff
@@ -204,4 +204,4 @@ jobs:
     needs: mvn-verify
     uses: ./.github/workflows/ci-ansible.yml
     with:
-      role: bfd-db-migrator
+      role: bfd-db-migrator
```

Comment:
No new line at EOF? ಠ_ಠ

Reply:
Added new Line

Reply:
Looks like it persists. Take a look at[ this resource](https://thoughtbot.com/blog/no-newline-at-end-of-file#following-the-rules-in-your-editor).

Reply:
I'm mistaken. Re-resolving 😉